src/main/java/com/dungeoncrawler/Javiarenka/dungeonMapGenerator/txt/stage.txt

```diff
@@ -0,0 +1,21 @@
+	 0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29
+0	 #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  # 
+1	 #        %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %     %  %  %  %  %  %  %  %  # 
+2	 #     #  #  #  #  #  #  #  %  #  #  #  #  %  #  #  #  #  #     #  #  #  #  #  %  #  %  # 
+3	 #     #                 #  %  #        #  %  #           .     #           #  %  #  %  # 
+4	 #     #                 #  %  #        #  %  #           #     .           #  %  #  %  # 
+5	 #     #                 #  %  #        #  %  #           #     #           #  %  #  %  # 
+6	 #     #  #  #  #  #  .  #  %  #  .  #  #  %  #  #  #  #  #     #  #  #  #  #  %  #  %  # 
+7	 #     %  %  %  %  %                                            %  %  %  %  %  %  %  %  # 
+8	 #     #  #  #  #  #     #  #     #  #  #  #  #  #  #  %  #  #  #  #  %  #  #  #  #  %  # 
+9	 #     #           #     #  #     #                 #  %  #        #  %  #        #  %  # 
+10	 #     #           .     #  #     #                 #  %  #        #  %  #        #  %  # 
+11	 #     #  #  #  #  #     #  #     #  #  .  #  #  #  #  %  #  .  #  #  %  #        .     # 
+12	 #                       %  %                                      #  %  #  #  #  #     # 
+13	 #     #  #  #  #  #  #  #  %  #  #  #     #  #  #  #  #  #  #                          # 
+14	 #     #                 #  %  #  #  #     .                 #  %  #  #  #  #  .  #  %  # 
+15	 #     .                 #  %  #  #  #     #                 #  %  #              #  %  # 
+16	 #  %  #                 #  %  #  #  #     #                 #  %  #              #  %  # 
+17	 #  %  #  #  #  #  #  #  #  %  #  #  #     #  #  #  #  #  #  #  %  #  #  #  #  #  #  %  # 
+18	 #  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  %  # 
+19	 #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  #  # 
```

Comment:
W razie czego adres tego typu plików możesz dodać do pliku .gitignore, żeby się nie wersjonowały za każdym razem.

Reply:
nice!

Reply:
Chyba nie rozumiem jak to działa. Dodałem takie linie:
### Dungeon Map Generator ###
**/dungeonMapGenerator/txt
**/dungeonMapGenerator/webStuff/debug

ale przy tworzeniu commitów ciągle bierze te pliki pod uwagę.

Reply:
a jak spróbujesz tak:

```
**/dungeonMapGenerator/txt
**/dungeonMapGenerator/webStuff/debug
```

Reply:
To samo. Tak właściwie to co ten .gitignore robi? Znaczy jakiego zachowania intelliJ miałbym się spodziewać? Że przy tworzeniu commitów on by nie chciał wysyłać, ani nie sprawdzał zmian w plikach z wymienionych folderów? Bo póki co, ciągle to robi...